CONTRIBUTING.md

```diff
@@ -1,11 +1,77 @@
-- [Setup](#setup)
-- [Development](#development)
-  - [Tooling](#tooling)
-  - [Commands](#commands)
-    - [Package Aliasing and Yarn Workspace](#package-aliasing-and-yarn-workspace)
-  - [Documentation](#documentation)
-  - [Storybook](#storybook)
-- [Making a Pull Request?](#making-a-pull-request)
-  - [Commit Convention](#commit-convention)
-  - [Steps to PR](#steps-to-pr)
-  - [Tests](#tests)
+# Contributing to Vitau-ui
+
+Here are a few guidelines that will help you along the way.
+
+## Branches
+
+### Branches naming standard
+
+[Contributor name]/[Project acronym]-[Jira task id]/-[task description]
+
+### Example
+
+```bash
+git checkout -b rulo/DSGN-001/select-component
+```
+
+- rulo = Contributor name
+- DSGN = Project acronym
+- 001 = Jira task id
+- select-component = task description
+
+## Commits
+
+### Commits naming standard
+
+[Jira task id] + [Package] [Verb] + [Substantive] + [What for]
+
+### Example
+
+```bash
+git commit -m '001-layout-Added flex layout for managing out templates'
+```
+
+- Jira task id - SSKB-1
+- Package - layout
+- Verb -  Added
+- Substantive - flex layout
+- What for - for managing out templates
+
+## Pull Requests
+
+### Before creating a pull request please
+
+- Make sure you created an Jira task related to it.
+- Make sure you're following the [branches section](#Branches) recommendations
+- Make sure you're following the [commits section](#Commits) recommendations
+
+
+### Steps to PR
+1- Create a local branch based off master
+```bash
+$ git checkout master
+$ git pull
+$ git checkout -b <branch-name>
+$ git push --set-upstream origin <branch-name>
+```
+
+2- When you've staged the changes, commit them.
+```bash
+$ git status
+$ git commit -m "The message goes here"
+```
+
+3- Share your branch.
+```bash
+$ git push origin <branch-name>
+```
+4- Submit a GitHub Pull Request and Get someone else to Code Review.
+
+5- After approved and all Circle CI checks have passed you can now merge the branch. Make sure you only have the commits you want to go into master
+
+```bash
+$ git checkout master
+$ git pull
+$ git merge <branch-name> --ff-only
+$ git push
+```
```

Comment:
Esto no es correcto porque no usamos CircleCI y tampoco debemos de hacer los merges asi, la convencion hasta ahorita es que los merges se hagan por github utilizando el modo `squash and merge`

Reply:
corregido